storage/client_raft_test.go

```diff
@@ -382,7 +389,7 @@ func TestFailedReplicaChange(t *testing.T) {
 
 	// The pending config change flag was cleared, so a subsequent attempt
 	// can succeed.
-	storage.TestingCommandFilter = nil
+	runFilter.Store(false)
```

Comment:
@kkaneda just merged something new that uses this filter, so it'll likely need an update.